providers/src/airflow/providers/edge/models/edge_worker.py

```diff
@@ -144,6 +144,12 @@ class EdgeWorker(BaseModel, LoggingMixin):
     sysinfo: str
     model_config = ConfigDict(from_attributes=True, arbitrary_types_allowed=True)
 
+    class SetStateReturn(BaseModel):
+        """Defines return type of set_state function."""
+
+        queues: Optional[List[str]] = None  # noqa: UP006,UP007 - prevent Sphinx failing
```

Comment:
Why is Sphinx failing here? Is it the same like in line 137?
Could we make this a top-level class and not a nested class (except if object is not needed according to commend above)?